modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeNumberInput.ts

```diff
@@ -25,8 +25,10 @@ const createBespokeNumberInput = (editor: Editor, backstage: UiFactoryBackstage,
   const onSetup = onSetupEvent(editor, 'NodeChange SwitchMode', (api: BespokeSelectApi) => {
     const comp = api.getComponent();
     currentComp = Optional.some(comp);
-    spec.updateInputValue(comp);
-    Disabling.set(comp, !editor.selection.isEditable());
+    if (comp.getSystem().isConnected()) {
```

Comment:
Why was this needed is this being executed to early?

Reply:
actually the problem seems to be that the component is composed by 3 components (minus, input and plus) but the "un-mounting" works only for the last one :( , so in our case the `plus`, I'm trying to figure out why is that :/

Reply:
That would explain why things are not working as expected when the setToolbar is executed twice. This could produce issues if the fontsizeinput is in the more drawer of the toolbar so that it gets rendered and re-rendered over and over so we need to fix this.

Reply:
ok, I implemented the fix that you suggested thanks for the help :)